site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,264 @@
+{{% /notice %}}
+
+{{% notice tip %}}
+When using AWS, you can use the host name supplied for the provisioned VM as your host ID, or use the `environment` host ID resolver and pass the host ID when launching the database process with an environment variable.
+{{% /notice %}}
+
+For example, if you used `M3DC_HOST_ID` for the environment variable name, use the following in your configuration:
+
+```yaml
+hostID:
+  resolver: hostname
+    - ${M3DC_HOST_ID:""}
+```
+
+Then start the `m3dbnode` process with:
+
+{{< tabs name="start_container" >}}
+{{% tab name="Pre-built binary" %}}
+
+```shell
+M3DB_HOST_ID=m3db001 m3dbnode -f <config-file.yml>
+```
+
+{{% notice info %}}
+Depending on your operating system setup, you might need to prefix the command with `sudo`.
+{{% /notice %}}
+
+{{% /tab %}}
+{{% tab name="Output" %}}
+
+<!-- TODO: Perfect image, pref with terminalizer -->
+
+<!-- TODO: Update image -->
+
+![Docker pull and run](/docker-install.gif)
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Kernel Configuration
+
+Depending on the default limits of your bare-metal machine or VM, M3 may need some Kernel tweaks to run as efficiently as possible, and [we recommend you review those](/docs/operational_guide/kernel_configuration) before running M3 in production.
+
+## Configuration files
+
+You configure each M3 component by passing the location of a YAML file with the `-f` argument.
+
+The file groups configuration into `coordinator` or `db` sections that represent the `M3Coordinator` and `M3DB` instances of single-node cluster.
+
+{{% notice tip %}}
+You can find more information on configuring M3DB in the [operational guides section](/docs/operational_guide/).
+{{% /notice %}}
+
+{{% notice note %}}
+The steps in this guide have the following 3 seed nodes, you need to change your configuration to suit the details of yours, including the details of an etcd cluster in the `etcdClusters` section of the M3 configuration file.
+{{% /notice %}}
+
+-   m3db001 (Region=us-east1, Zone=us-east1-a, Static IP=10.142.0.1)
+-   m3db002 (Region=us-east1, Zone=us-east1-b, Static IP=10.142.0.2)
+-   m3db003 (Region=us-east1, Zone=us-east1-c, Static IP=10.142.0.3)
+
+### M3DB node
+
+[Start with the M3DB configuration template](https://github.com/m3db/m3/blob/master/src/dbnode/config/m3dbnode-cluster-template.yml) and change it to suit your cluster.
+
+The example below connects to an etcd instance in a zone called `eu-1`
+This example updates the `service` and `seedNodes` sections to match the node details above:
+
+<!-- TODO: Add more details on config items here -->
+
+```yaml
+config:
+  service:
+    env: default_env
+    zone: embedded
+    service: m3db
+    cacheDir: /var/lib/m3kv
+    etcdClusters:
+      - zone: embedded
+        endpoints:
+          - 10.142.0.1:2379
+          - 10.142.0.2:2379
+          - 10.142.0.3:2379
```

Comment:
This now should be (with v1.0.0):
```
 discovery: 
   type: m3db_cluster 
   m3dbCluster: 
     env: default_env
     endpoints: 
       - 10.142.0.1:2379
       - 10.142.0.2:2379
       - 10.142.0.3:2379
```

As per example:
https://github.com/m3db/m3/blob/ec68e83b38a7834083a839558fbfaae8c7d61a38/scripts/docker-integration-tests/dedicated_etcd_embedded_coordinator/m3dbnode.yml#L25-L31

We should also update the template to match:
https://github.com/m3db/m3/blob/master/src/dbnode/config/m3dbnode-cluster-template.yml

Reply:
@robskillington Just to understand, replace the whole config, or just the `service` part down? Or just the `endpoints`?

Reply:
Ah sorry, just service downwards.